### PR TITLE
Feat : Add Signup page

### PR DESCRIPTION
**Pull request by MaryamAK7**

### CODE REVIEW

**Comment by MaryamAK7** [comment]
Make sure to use a 6 digit password not less, the local storage will be added later when I finish all the authentication files.

File: src/App.js [hunk 1]
```diff
@@ -8,50 +8,56 @@ import AboutUsPage from './Pages/AboutUs/AboutUs';
 import SingleCityPage from './Pages/SingleCityPage/SingleCityPage';
 import CitiesPage from './Pages/CitiesPage/CitiesPage';
 import cities from './MockData/cities.json';
-import ErrorPage from './Pages/ErrorPage/ErrorPage';
+// import ErrorPage from './Pages/ErrorPage/ErrorPage';
```
Comment (osamaakb) [inline src/App.js:11]: put back the comment

File: src/Pages/SignUpPage/SignUpPage.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
+import React, { useState, useRef } from 'react';
+import { useAuth } from '../../Context/AuthContext.js';
+import { Link, useHistory } from 'react-router-dom';
+
+export default function SignUpPage() {
```
Comment (osamaakb) [inline src/Pages/SignUpPage/SignUpPage.js:5]: I was thinking more of adding a sign-in with google, it's easier to implement and easier for users too.
take a look here
https://github.com/Lebanon-WBC-Capstones/e-track-web/pull/86/files

**Comment by osamaakb** [comment]
also fix the conflicts

**Pull request closed without merging**